dist/utilities/border/_border.scss

```diff
@@ -6,12 +6,12 @@
 
 @mixin border(
     $important: false,
-    $border: if(variable-exists(border), $border, null)
```

Comment:
I'm not quite convinced this works well for the user. This is actually taking options away from the user.

For one, `$border-color` is used in the original `$border` variable, so modifying one modifies the other.

Secondly, by hardcoding the border width and style, the developer can no longer customize that appearance.

So, unless you provide the ability to override the default width and style, this mixin is becoming too restrictive imo.